components/event-sources/adapter/http/adapter.go

```diff
@@ -0,0 +1,196 @@
+package http
+
+import (
+	"context"
```

Comment:
Grouping imports would be nice, at the very least separate stdlib from third-parties.